collector/utils.py

```diff
@@ -13,10 +15,22 @@
 # pylama:ignore=E1101
 requests.packages.urllib3.disable_warnings()
 
+if not (REDIS_ENV and REDIS_PASSWORD):
+    LOGGER.error('Environment not set properly for Redis')
+    sys.exit(Arbiter.APP_LOAD_ERROR)
 
 REDIS = redis.Redis(**json.loads(REDIS_ENV), password=REDIS_PASSWORD)
 
 
+def ping_redis() -> str:
+    """Call ping on Redis."""
+    try:
+        return REDIS.ping()
```

Comment:
you want to change return type to `True/False`?

Reply:
Yes, will change it to bool
Thanks.